src/Symfony/Contracts/Deprecation/README.md

```diff
@@ -15,8 +16,11 @@ The function requires at least 3 arguments:
 
 Example:
 ```php
-deprecated('symfony/blockchain', 8.9, 'Using "%s" is deprecated, use "%s" instead.', 'bitcoin', 'fabcoin');
+trigger_deprecation('symfony/blockchain', 8.9, 'Using "%s" is deprecated, use "%s" instead.', 'bitcoin', 'fabcoin');
```

Comment:
One problem has been brought up that affects the version: locale-aware casts from float to string. This example should pass the version on string. For this effect, I would show people to not rely on it until the locale-awareness on casts has been removed in PHP.

Reply:
updated.